<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1D1D1D;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #F11D44;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lb_tentaikhoan">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:black;
border-radius:12px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_home">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Icon &amp; image/Logo-Transparent-01.png</normaloff>../Icon &amp; image/Logo-Transparent-01.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Icon &amp; image/2203523_arrow_back_botton_left_icon.ico</normaloff>../Icon &amp; image/2203523_arrow_back_botton_left_icon.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>261</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#0F151A;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_dangxuat">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>580</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;</string>
      </property>
      <property name="text">
       <string>Đăng Xuất</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_trangchu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>51</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Trang Chủ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_qlphong">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Quản lý phòng</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_qlmay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Quản lý máy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tbcsvc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>TB Và CSVC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dichvu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Dịch vụ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_nhanvien">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Nhân viên</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tk">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
background-color:#F11D44;</string>
      </property>
      <property name="text">
       <string>Tài khoản</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tknhanvien">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>460</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;
color:white;
border-left:none;
border-top:none;
border-right:none;
</string>
      </property>
      <property name="text">
       <string>Tài khoản nhân viên</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/Test 1-01.png</normaloff>../../Downloads/Test 1-01.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QTableWidget" name="tb_taikhoan">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>90</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tb_taikhoan_ls">
     <property name="geometry">
      <rect>
       <x>636</x>
       <y>90</y>
       <width>341</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>50</y>
       <width>21</width>
       <height>581</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #F11D44;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_timkiem">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>640</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
border:none;
color:black;
background-color:#fff;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_timkiem">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>640</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
border:none;
color:black;
background-color:#fff;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tb_taikhoan_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>370</y>
       <width>361</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:White;</string>
     </property>
     <property name="text">
      <string>Lịch sử cập nhật</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:White;</string>
     </property>
     <property name="text">
      <string>Lịch sử nạp </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:White;</string>
     </property>
     <property name="text">
      <string>Thông tin chi tiết</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1471</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
